src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,10 +34,40 @@ class AssetsInstallCommand extends ContainerAwareCommand
     const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
+    private $baseDir;
+    private $filesystem;
+
     /**
-     * @var Filesystem
+     * @param string          $baseDir
+     * @param Filesystem|null $filesystem
      */
-    private $filesystem;
+    public function __construct($baseDir = null, Filesystem $filesystem = null /**, $newApi = false*/)
```

Comment:
@GuilhemN `Application::getKernel()` returns `KernelInterface` which does not provide `getProjectDir()`. I'd keep injecting it just to avoid a call to `getContainer()->getParameter(...)`.